common/djangoapps/enrollment/views.py

```diff
@@ -616,13 +616,14 @@ def post(self, request):
                 org = course_id.org
                 update_email_opt_in(request.user, org, email_opt_in)
 
+            log.info('The user %s has already been enrolled in course run %s.', username, course_id)
```

Comment:
Why did you think the user has been enrolled into the course run by this point of the code? After reading the `if...else` block above, I think here means a enrollment status modification was made. But it is unclear what modification it is.